cloudify/utils.py

```diff
@@ -242,7 +242,9 @@ def get_execution_token():
     Returns the execution token to use when calling the REST service
     """
     try:
-        return _get_current_context().execution_token
+        context = _get_current_context()
+        return context.execution_token if hasattr(context, 'execution_token') \
```

Comment:
hmm i dont understand, why would there be no attribute? there is a property now, so the hasattr should have no reason to be here?

Reply:
is it for WorkflowContext? maybe we also need to add the property there?

Reply:
Ah yes I added it because it was failing when the context was `CancelCloudifyContext`, I thought I shouldn't added it there, but maybe I do?